TestResultSummaryService/routes/deleteBuildData.js

```diff
@@ -9,7 +9,7 @@ module.exports = async (req, res) => {
         return res.status(400).send({ message: "BuildUrl is missing" });
     } else {
         const buildMonitor = new BuildMonitor();
-        const task = { buildUrl, numBuildsToKeep: 0 };
+        const task = { buildUrl, numBuildsToKeep: 0, deleteForever: true };
```

Comment:
- Could we use the same name `keepForever `? Switching between `keepForever ` and `deleteForever ` is a little confusing.
- we should allow users to set this value, not hardcode it.